src/redux/reducers/messages.js

```diff
@@ -0,0 +1,20 @@
+const initState = '';
+
+const messageReducer = (state = initState, action) => {
+  switch (action.type) {
+    case 'COMPLETE_MESSAGE':
+      return action.payload.message;
+    default:
+      return state;
+  }
+};
+
+export const fetchmessage = () => async (dispatch) => {
+  await fetch('http://127.0.0.1:3020/')
+    .then((response) => response.json())
```

Comment:
Kindly change the url, inside fetch method to point to: "http://localhost:3000".

### :slightly_smiling_face: